src/mock/product/index.ts

```
@@ -0,0 +1,99 @@
+  imageAlt: string;
+  category: string;
+}
+
+export const productData: ProductData[] = [
+  {
+    id: "1",
+    title: "ChatGPT Plus",
+    price: "월 20,000원",
+    imageUrl: "/image/product/chatgpt.png",
+    imageAlt: "ChatGPT Plus",
+    category: "ai"
```

ESLint

[error] Insert `,`

(prettier/prettier)